spec/models/ems_event_spec.rb

```diff
@@ -309,6 +309,29 @@
     end
   end
 
+  describe ".event_groups (no stubbing)" do
```

Comment:
What does the "no stubbing" signify?

Reply:
The old code didn't do stub_settings_merge either, so I'm not sure why the need to add a qualifier.  I'm not against it, it just seems unnecessary?

Reply:
Perhaps this is better?

```suggestion
  # NOTE: Do not use Settings stubs here (e.g. stub_settings_merge), as this test is meant to
  # test the actual Settings across all providers.
  describe ".event_groups" do
```

Reply:
ohhhhh wait, I misunderstood - you're changing the name so it doesn't collide with the _other_ context entry below with the same name.

Ok, then in that case, let's signify that this is testing the "real" settings with the comment _and_ a qualifier...maybe something like:

```suggestion
  # NOTE: Do not use Settings stubs here (e.g. stub_settings_merge), as this test is meant to
  # test the actual Settings across all providers.
  describe ".event_groups (actual Settings)" do
```

Reply:
> ohhhhh wait, I misunderstood - you're changing the name so it doesn't collide with the other context entry below with the same name.

Exactly.  They belong in the same context but that one is stubbing the settings which we explicitly don't want to do here.

Reply:
I'll make those changes.  Thanks.